Podcast/ViewController.swift

```diff
@@ -29,6 +29,7 @@ class ViewController: UIViewController {
         guard let appDelegate = UIApplication.shared.delegate as? AppDelegate, let mainScrollView = mainScrollView else { return }
         if appDelegate.tabBarController.accessoryViewController == nil {
             mainScrollView.contentInset.bottom = appDelegate.tabBarController.tabBarHeight
+            mainScrollView.scrollIndicatorInsets = UIEdgeInsetsMake(0, 0, 0, appDelegate.tabBarController.tabBarHeight)
```

Comment:
Does this work? If not, let's uncommit this file.

Reply:
Hmm yeah I don't think this is what we want.

Reply:
lol nope it doesn't, i'll undo that